plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerOptions.java

```diff
@@ -73,6 +73,9 @@
    */
   private static final long STALE_STATE_DELAY_MS = 30000;
 
+  // TODO docs
+  private static final int REFRESH_INTERVAL_IN_MILLIS_DEFAULT = 1;
```

Comment:
I think we can make the default refresh rate of 60FPS? That'd require a delay of 16.67 ms. Let's test this out though and decide more experimentally.

Reply:
Sounds good, I'll update the variable and we can test it out